compiler/shared_ast/typing.ml

```diff
@@ -117,7 +110,7 @@ let rec format_typ
   | TStruct s -> Format.fprintf fmt "%a" A.StructName.format_t s
   | TEnum e -> Format.fprintf fmt "%a" A.EnumName.format_t e
   | TOption t ->
-    Format.fprintf fmt "@[<hov 2>%a@ %s@]" format_typ_with_parens t "eoption"
+    Format.fprintf fmt "@[<hov 2>(%a)@ %s@]" format_typ_with_parens t "eoption"
```

Comment:
Can #450 help us here?

Reply:
I don't think #450 changes the printer for types, so no